### ISSUE TITLE
Test decorations on gutter seem to have low contrast in Light High Contrast Theme 

### ISSUE THREAD

**Issue by luabud** [microsoft/vscode#143835]
Testing #143425

If you open the test explorer and run tests that fail, the error icons on the gutter seem to have low contrast:

![image](https://user-images.githubusercontent.com/45497113/155433188-2b10f270-d886-454e-abae-40fd4736bb42.png)

Same for when the tests pass:

![image](https://user-images.githubusercontent.com/45497113/155433237-fde67ed4-a2db-42fa-b340-04f8dd1a8689.png)

Here's a repo that has tests enabled and configured: https://github.com/luabud/sample_project

**Comment by daviddossett** [comment]
Matched the colors used for success/error in other areas of the light HC theme.

For some reason it looks like these colors get somewhat dimmed in certain cases. Doesn't look related to this theme but I'll dig a little deeper and see if I can find out what's happening. 

<img width="368" alt="CleanShot 2022-03-07 at 10 17 31@2x" src="https://user-images.githubusercontent.com/25163139/157093839-62f26866-4d9e-4c0f-a5a5-3292ff6e36df.png">

**Comment by daviddossett** [comment]
Subtle, but here's an example. It's ever-so-slightly faded before the tests have been run.

### Before running tests
<img width="263" alt="CleanShot 2022-03-07 at 10 19 15@2x" src="https://user-images.githubusercontent.com/25163139/157094128-9ade5d27-dc1b-42b7-989c-fdb1d85f1e2e.png">

### After running tests
<img width="266" alt="CleanShot 2022-03-07 at 10 21 15@2x" src="https://user-images.githubusercontent.com/25163139/157094377-a5009292-a5a5-4d46-9422-19578783b5b5.png">

**Comment by connor4312** [comment]
Yes, tests run before the current window are 'outdated' and are faded slightly. There used to be way for extensions to mark tests as outdated, but this is no longer a thing, so we could get rid of the outdated notion.

**Comment by daviddossett** [comment]
> Yes, tests run before the current window are 'outdated' and are faded slightly. There used to be way for extensions to mark tests as outdated, but this is no longer a thing, so we could get rid of the outdated notion.

Ok great, thanks. Looks like removing this did the trick. I can push this if that's all to consider here:

https://github.com/microsoft/vscode/blob/d4e28503c4f8c9931ca850774abc038fa857e9e4/src/vs/workbench/contrib/testing/browser/media/testing.css#L68-L71

**Comment by connor4312** [comment]
Please leave it for now, I'll clean it up as part of https://github.com/microsoft/vscode/issues/144618

**Comment by daviddossett** [comment]
Sounds good, thank you!
